tests/units/feature_extraction/test_feature_calculations.py

```diff
@@ -72,6 +73,13 @@ def assertEqualPandasSeriesWrapper(self, f, input_to_f, result, *args, **kwargs)
                          msg="Not equal for pandas.Series: %s != %s" % (
                              f(pd.Series(input_to_f), *args, **kwargs), result))
 
+    def test__roll(self):
+        np.random.seed(42)
+        x = np.random.normal(size=30)
+        for shift in [1, 10, 30, 50, 150]:
```

Comment:
would also add odd and negative shift parameters, maybe use [-7, -1, 0, 1, 12, 511]